client/src/index.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
+import { Route, BrowserRouter as Router } from 'react-router-dom'
+import Profile from './Components/Pages/Profile';
+import Experts from './Components/Pages/Experts';
+import Learners from './Components/Pages/Learners';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+const routing = (
+  <Router>
+    <div>
+      <Route exact path="/" component={App} />
+      <Route path="/profile" component={Profile} />
+      <Route path="/experts" component={Experts} />
+      <Route path="/learners" component={Learners} />
+    </div>
```

Comment:
- Use `Switch` so you don't need to pass `exact` as props.
- Remove `<div> </div>`